modules/tensor_mechanics/test/tests/elasticitytensor/rotation_matrix_2_rotations.i

```diff
@@ -0,0 +1,223 @@
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./C1133_aux_euler]  # C13
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./C1112_aux_euler]  # C16
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+[]
+
+[AuxKernels]
+  [./matl_C1111_matrix]  # C11
+    type = RankFourAux
+    rank_four_tensor = rotation_matrix_elasticity_tensor
+    index_i = 0
+    index_j = 0
+    index_k = 0
+    index_l = 0
+    variable = C1111_aux_matrix
+    execute_on = initial
+  [../]
+  [./matl_C1122_matrix]  # C12
+    type = RankFourAux
+    rank_four_tensor = rotation_matrix_elasticity_tensor
+    index_i = 0
+    index_j = 0
+    index_k = 1
+    index_l = 1
+    variable = C1122_aux_matrix
+    execute_on = initial
+  [../]
+  [./matl_C1133_matrix]  # C13
+    type = RankFourAux
+    rank_four_tensor = rotation_matrix_elasticity_tensor
+    index_i = 0
+    index_j = 0
+    index_k = 2
+    index_l = 2
+    variable = C1133_aux_matrix
+    execute_on = initial
+  [../]
+  [./matl_C1112_matrix]  # C16
+    type = RankFourAux
+    rank_four_tensor = rotation_matrix_elasticity_tensor
+    index_i = 0
+    index_j = 0
+    index_k = 0
+    index_l = 1
+    variable = C1112_aux_matrix
+    execute_on = initial
+  [../]
+
+  [./matl_C1111_euler]  # C11
+    type = RankFourAux
+    rank_four_tensor = euler_elasticity_tensor
+    index_i = 0
+    index_j = 0
+    index_k = 0
+    index_l = 0
+    variable = C1111_aux_euler
+    execute_on = initial
+  [../]
+  [./matl_C1122_euler]  # C12
+    type = RankFourAux
+    rank_four_tensor = euler_elasticity_tensor
+    index_i = 0
+    index_j = 0
+    index_k = 1
+    index_l = 1
+    variable = C1122_aux_euler
+    execute_on = initial
+  [../]
+  [./matl_C1133_euler]  # C13
+    type = RankFourAux
+    rank_four_tensor = euler_elasticity_tensor
+    index_i = 0
+    index_j = 0
+    index_k = 2
+    index_l = 2
+    variable = C1133_aux_euler
+    execute_on = initial
+  [../]
+  [./matl_C1112_euler]  # C16
+    type = RankFourAux
+    rank_four_tensor = euler_elasticity_tensor
+    index_i = 0
+    index_j = 0
+    index_k = 0
+    index_l = 1
+    variable = C1112_aux_euler
+    execute_on = initial
+  [../]
+[]
+
+[Materials]
+  [./elasticity_matrix]
+    type = ComputeElasticityTensor
+    block = 0
+    base_name = 'rotation_matrix'
+    fill_method = symmetric9
+    C_ijkl = '1111 1122 1133 2222 2233 3333 2323 1313 1212'
+    # rotation matrix for rotating a vector
+    #   1. 45 degrees about z-axis
+    #   2. ~54.7 degrees (arccos(1/sqrt(3)) radians) about x-axis
+    # then taking the tranpose to give sample-to-crystal rotation,
+    # ie. R*([0,0,1]) = [1,1,1], meaning the <001> direction of the sample
+    # (or simulation) frame points along the <111> direction of the crystal
+    rotation_matrix = '0.70710678  0.40824829  0.57735027
+                      -0.70710678  0.40824829  0.57735027
+                       0.         -0.81649658  0.57735027'
+  [../]
```

Comment:
This is nice and particularly helpful example of how to get the loading directions in the crystal and simulation frames to match (e.g. 111 is a common loading direction).

Would you be willing to expand the documentation to include the two examples here that you've added in the test input files

Reply:
I suppose I could do that. We'll see how concise we can make it. The documentation for `ComputeElasticityTensor` is already getting a bit long.

Reply:
I mean the _webpage_ is getting a bit long, though I suppose more information is better than less.

Reply:
@sapitts I added the examples to the documentation for `ComputeElasticityTensor` in the most recent commit. Hopefully it is concise enough and didn't confuse matters further.